### PR TITLE
Norman

### PR DESCRIPTION
**Pull request by aaron-oluk**
new dashboard

### CODE REVIEW

**Review by aaron-oluk — COMMENTED** [review]
this is approved

File: appstarter/.htaccess [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<IfModule !mod_rewrite.c>
+	# If we don't have mod_rewrite installed, all 404's
+	# can be sent to index.php, and everything works as normal.
+	ErrorDocument 404 index.php
+</IfModule>
+
+# Disable server signature start
+	ServerSignature Off
+# Disable server signature end
```
Comment (Leon12-aaryson) [inline appstarter/.htaccess:49]: aprove all code for codeignitor

File: appstarter/app/Config/Routes.php [hunk 2]
```diff
@@ -30,6 +30,11 @@
 // We get a performance increase by specifying the default
 // route since we don't have to scan directories.
 $routes->get('/', 'Home::index');
+$routes->get('employee', 'Home::Employee');
+$routes->get('add-employee', 'Home::CreateEmployee');
+$routes->get('department', 'Home::Department');
+$routes->get('branch-master', 'Home::BranchMaster');
+
```
Comment (Leon12-aaryson) [inline appstarter/app/Config/Routes.php:37]: this is for the paths

File: appstarter/app/Controllers/Home.php [hunk 3]
```diff
@@ -6,6 +6,29 @@ class Home extends BaseController
...
+    public function Department()
+    {
+        return view("department");
+    }
+
+    public function BranchMaster()
+    {
+        return view("branch-master");
+    }
+
```
Comment (Leon12-aaryson) [inline appstarter/app/Controllers/Home.php:32]: functions

File: appstarter/app/Views/branch-master.php [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                        </table>
+                    </div>
+                </div>
+               </div>
+
+            </div>
+        </main>
+        <!-- html for main part ends here -->
+
+<?= $this->endSection() ?>
```
Comment (Leon12-aaryson) [inline appstarter/app/Views/branch-master.php:94]: tables

File: appstarter/app/Views/create-employee.php [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+            <a href="#" class="btn btn-sm" style="background: var(--skin-color); color: #fff">Create</a>
+                </div>
+
+            </div>
+        </main>
+        <!-- html for main part ends here -->
+
+    </section>
+
+<?= $this->endSection() ?>
```
Comment (Leon12-aaryson) [inline appstarter/app/Views/create-employee.php:233]: dashboard

File: appstarter/env [hunk 6]
```diff
@@ -30,13 +30,13 @@
...
+database.default.hostname = localhost
+database.default.database = hrmsdb
+database.default.username = root
+database.default.password = 
+database.default.DBDriver = MySQLi
+database.default.DBPrefix =
+database.default.port = 3306
 
 # database.tests.hostname = localhost
 # database.tests.database = ci4_test
```
Comment (aaron-oluk) [inline appstarter/env:42]: env

File: appstarter/public/assets/css/style.css
Comment (aaron-oluk) [inline appstarter/public/assets/css/style.css:1]: approve styling

File: appstarter/public/assets/img/default-avatar.png
Comment (aaron-oluk) [inline appstarter/public/assets/img/default-avatar.png:1]: png file

File: appstarter/public/assets/css/skin/color-1.css
Comment (aaron-oluk) [inline appstarter/public/assets/css/skin/color-1.css:1]: color css

File: appstarter/env
Comment (aaron-oluk) [inline appstarter/env:1]: env

**Review by aaron-oluk — COMMENTED** [review]
new

**Pull request merged**